LEAF_Request_Portal/templates/form.tpl

```diff
@@ -18,11 +18,12 @@
                         <!-- <button id="prevQuestion">Previous question</button> -->
                         <!-- <button class="button" dojoType="dijit.form.Button" onclick="checkForm(false);"><div id="save_indicator">Save Change</div></button> -->
                     </div>
+                    <input type="submit" value="Submit" aria-disabled="true" hidden>
                 </form>
             </div>
             <div id="progressArea2" style="height: 34px; background-color: #feffd2; padding: 4px; border-top: 1px solid black">
                 <div style="float: left"><button id="prevQuestion" type="button" class="buttonNorm prevQuestion"><img src="../libs/dynicons/?img=go-previous.svg&amp;w=22" alt="Previous" /> Previous Question</button></div>
```

Comment:
We should also include an aria-label for going back to the previous question. Something like `aria-label="Previous"`.